.github/workflows/build-docker.yml

```diff
@@ -44,3 +45,34 @@ jobs:
           push: true
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+
+  build-and-push-scs-compliance-check-image:
```

Comment:
Just FYI: this key seems to be a duplicate of the one in line 18

Reply:
Good catch! I changed the job name.